CanonicalLtd/maas-docs
# Remove media/ folder

Instead of including images in the local `./media/` folder, link to those assets on our assets server - https://assets.ubuntu.com/. Also remove the `.//media/` folder completely, and change the build commands to not look for the folder.

In future, if you want to add images, upload them through https://manager.assets.ubuntu.com/ and link to the image on assets.ubuntu.com.

This is in preparation for migrating these docs over to be discourse-based.

QA
--

``` bash
./run
```

And go to http://localhost:8203/2.5/en/intro-explore - browse around and check images still load okay.
